src/Security/Authentication/Negotiate/src/NegotiateHandler.cs

```diff
@@ -57,7 +57,7 @@ protected new NegotiateEvents Events
         /// <returns></returns>
         protected override Task<object> CreateEventsAsync() => Task.FromResult<object>(new NegotiateEvents());
 
-        private bool IsHttp2 => string.Equals("HTTP/2", Request.Protocol, StringComparison.OrdinalIgnoreCase);
+        private bool IsSupportedProtocol => HttpProtocols.IsHttp10(Request.Protocol) || HttpProtocols.IsHttp11(Request.Protocol);
```

Comment:
I inverted this logic to future proof it for HTTP/3. Better to allow-list than deny-list